plugins/cue_points/base/lib/kCuePointManager.php

```diff
@@ -1009,7 +1013,8 @@ public static function reIndexCuePointEntry(CuePoint $cuePoint, $shouldReIndexTo
 	public static function copyCuePointsToClipEntry( entry $clipEntry ) {
 		$clipAtts =  self::getClipAttributesFromEntry( $clipEntry );
 		//if clipConcat flow let batch job copy cue point
-		if ( !is_null($clipAtts) &&  is_null($clipEntry->getClipConcatTrimFlow()) ) {
+		$allowedFlows = array(EntryFlowType::CLIP_CONCAT, EntryFlowType::TRIM_CONCAT);
+		if ( !is_null($clipAtts) &&  !in_array($clipEntry->getFlowType(), $allowedFlows)) {
```

Comment:
instead of !is_null($clipAtts) , if($clipAtts) does the same and more readable , please change